src/Wallabag/CoreBundle/Resources/translations/messages.fr.yml

```diff
@@ -1,422 +1,422 @@
-            filter: 'Filtrer'
+            clear: "Effacer"
+            filter: "Filtrer"
     view:
         left_menu:
-            back_to_top: 'Revenir en haut'
-            back_to_homepage: 'Retour'
-            set_as_read: 'Marquer comme lu'
-            set_as_unread: 'Marquer comme non lu'
-            set_as_starred: 'Mettre en favori'
-            view_original_article: 'Article original'
-            re_fetch_content: 'Recharger le contenu'
-            delete: 'Supprimer'
-            add_a_tag: 'Ajouter un tag'
-            share_content: 'Partager'
-            share_email_label: 'Email'
-            public_link: 'Lien public'
-            delete_public_link: 'Supprimer lien public'
-            download: 'Télécharger'
-            print: 'Imprimer'
+            back_to_top: "Revenir en haut"
+            back_to_homepage: "Retour"
+            set_as_read: "Marquer comme lu"
+            set_as_unread: "Marquer comme non lu"
+            set_as_starred: "Mettre en favori"
+            view_original_article: "Article original"
+            re_fetch_content: "Recharger le contenu"
+            delete: "Supprimer"
+            add_a_tag: "Ajouter un tag"
+            share_content: "Partager"
+            share_email_label: "Courriel"
+            public_link: "Lien public"
+            delete_public_link: "Supprimer lien public"
+            download: "Télécharger"
+            print: "Imprimer"
             problem:
-                label: 'Un problème ?'
-                description: "Est-ce que cet article s'affiche mal ?"
-        edit_title: 'Modifier le titre'
-        original_article: 'original'
-        annotations_on_the_entry: '{0} Aucune annotation|{1} Une annotation|]1,Inf[ %count% annotations'
-        created_at: 'Date de création'
+                label: "Un problème ?"
+                description: "Est-ce que cet article s’affiche mal ?"
+        edit_title: "Modifier le titre"
+        original_article: "original"
+        annotations_on_the_entry: "{0} Aucune annotation|{1} Une annotation|]1,Inf[ %count% annotations"
+        created_at: "Date de création"
     new:
-        page_title: 'Sauvegarder un nouvel article'
-        placeholder: 'http://website.com'
+        page_title: "Sauvegarder un nouvel article"
+        placeholder: "http://website.com"
         form_new:
-            url_label: Url
+            url_label: "Adresse"
     edit:
-        page_title: 'Éditer un article'
-        title_label: 'Titre'
-        url_label: 'Url'
-        is_public_label: 'Public'
-        save_label: 'Enregistrer'
+        page_title: "Éditer un article"
+        title_label: "Titre"
+        url_label: "Adresse"
+        is_public_label: "Public"
+        save_label: "Enregistrer"
     public:
-        shared_by_wallabag: "Cet article a été partagé par <a href='%wallabag_instance%'>wallabag</a>"
+        shared_by_wallabag: "Cet article a été partagé par <a href=’%wallabag_instance%’>wallabag</a>"
```

Comment:
Nope, please keep `'` inside this line, it's not a french phrase, it's plain HTML